functions/system.bash

```diff
@@ -554,3 +554,31 @@ prepare_serial_port() {
 
   whiptail --title "Operation successful" --msgbox "$successText" 9 80
 }
+
+
+## Function for installing the ELElabs EZSP firmware update utility
+## Useful to flash various ZigBee controllers that have an Ember chipset.
+##
+##    ezspUtility_setup
+##
+##
+ezspUtility_setup() {
+  local repo=https://github.com/Elelabs/elelabs-zigbee-ezsp-utility
+  local destdir=/usr/share/openhab
+  local target=Elelabs_EzspFwUtility.py
+  local successText="The ELElabs python tool to flash Ember chipset ZigBee controllers has been successfully installed as ${destdir}/${target}.\\n\\nSee https://github.com/Elelabs/elelabs-zigbee-ezsp-utility#how-to how to use it."
+
+
+  temp="$(mktemp -d "${TMPDIR:-/tmp}"/openhabian.XXXXX)"
+
+  echo -n "$(timestamp) [openHABian] Installing ELElabs firmware flash tool ... "
+  cond_redirect git clone "$repo" "$temp"
+  git clone https://github.com/xsp1989/zigbeeFirmware.git /tmp/zbfw
```

Comment:
Why clone it into `/tmp` why not just clone both to `/opt/elelabs` and then on the second run update the repos. See the FireMotD install for an example.

Reply:
I think I got interrupted or some such. I don't want to get the firmware from that 2nd repo in all cases so I just added the URL only (no cloning) now just to remember